.deepsource.toml

```diff
@@ -15,6 +15,10 @@
 #
 version = 1
 
+exclude_patterns = [
+  "**/*.pb.go"
```

Comment:
could you please exclude example dir & _test.go & _mock_test.go & bench_test.go file ?

Reply:
I think I need some discussion in team about excluding `_test.go` files as I think it may affect our testing code quality.

Reply:
I see. okay, please exclude example dir